CHANGELOG.asciidoc

```diff
@@ -45,7 +45,7 @@
   * Fix bug in OS detection
   
 == 0.0.18 (May 6, 2016)
-  * Support newer Facter api (Aaron Mildtenstein)
+  * Support newer Facter api (Aaron Mildenstein)
```

Comment:
Not sure this should be fixed :P

Reply:
https://github.com/jordansissel/pleaserun/blame/7621b9889b79025364ee52fb7b58f4abf2dc7b75/CHANGELOG.asciidoc#L48
https://github.com/jordansissel/pleaserun/blame/7621b9889b79025364ee52fb7b58f4abf2dc7b75/CHANGELOG.asciidoc#L51

https://www.google.com/search?q=Aaron+Mildtenstein
> Showing results for Aaron Mildenstein

Reply:
@untergeek